components/dashboard/BasicUserInfo/UI/UserOneLiner.tsx

```diff
@@ -0,0 +1,37 @@
+'user client';
+import React from 'react';
+import styled from 'styled-components';
+
+const UserOneLiner = () => {
+  return (
+    <UserOneLinerContainer>
+      <UserNameTitle>한 줄 소개</UserNameTitle>
+      <UserOneLinerTextarea placeholder="한 줄 소개를 입력해주세요"></UserOneLinerTextarea>
+    </UserOneLinerContainer>
+  );
+};
+
+export default UserOneLiner;
+
+const UserOneLinerTextarea = styled.textarea`
+  gap: 1rem;
+  align-items: flex-start;
+  border: 0.1rem solid ${({ theme }) => theme.colors.grey_600};
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.colors.grey_600};
```

border는 px써서 고정해줍시다!